README.md

```diff
@@ -435,7 +435,7 @@ The following optional parameters are supported:
 - `annotations` - object containing annotations for the action (default: `{}`)
 - `limits` - object containing limits for the action (default: `{}`)
 - `kind` - runtime environment parameter, ignored when `action` is an object (default: `nodejs:default`)
-- `version` - set semantic version of the action. If parameter is empty when create new action openwisk generate 0.0.1 value when update an action increase the patch version.
+- `version` - set semantic version of the action. If parameter is empty when create new action openwhisk generate 0.0.1 value when update an action increase the patch version.
```

Comment:
Grammar needs more fixing here and casing for ow should be OpenWhisk.  

Separately we should remove the reference to version. It is incorrect entirely.